### ISSUE TITLE
Can't find declaration files after switch to macro

### ISSUE THREAD

**Issue by irrigator** [emotion-js/emotion#933]
- `emotion` version: 9.2.12
- `react` version: 16.5.2

Relevant code.

```javascript
import styled from 'react-emotion/macro'

const H1 = styled.h1({
  fontSize: 48,
  textAlign: 'center',
})
```

What you did:

Using `react-emotion/macro` in create-react-app.

What happened:

Visual Studio Code can't find typescript declaration file and autocompletion stops working.

**Comment by Ailrun** [comment]
Thank you for reporting this issue!
We currently don't have typings for `macro` files.
I will add types for those files in emotion@10, after fixing some other issues.

**Comment by irrigator** [comment]
Thanks for your effort.

**Comment by Ailrun** [comment]
No, it's not yet provided. I'm recently too busy to update one. If you want to send a PR for this, feel free to send it and we will welcome.

**Comment by slikts** [comment]
Updated workaround:
```ts
declare module '@emotion/styled/macro' {
  import styled from '@emotion/styled';
  export * from '@emotion/styled';
  export default styled;
}
```

**Comment by osdiab** [comment]
It seems this hasn't been published yet - any plans on a release containing this?

**Comment by peduarte** [comment]
Has this been published? I'm still getting the error :/

**Comment by SavePointSam** [comment]
As of [@emotion/styled v10.0.11](https://github.com/emotion-js/emotion/tree/eec0e9502802ff234e93932cf6e1a4f7dd1b02df/packages/styled) (current version), the file added in #1315 (macro.d.ts) is not available as it has not been [whitelisted in the package.json](https://github.com/emotion-js/emotion/blob/24c66c194f5120ad1d6b9e9764bd59ed9c38bd95/packages/styled/package.json#L23). So it does not appear in the current build available on NPM.

It is also curious that v10.0.11 doesn't appear in the GitHub releases panel, only on NPM and inside the CHANGELOG.md for @emotion/styled.

I've created a PR to fix this (#1369).

For future reference, you can browse the latest release [here](https://runpkg.com/?@emotion/styled@latest) and browse v10.0.11 specifically [here](https://runpkg.com/?@emotion/styled@10.0.11).

**Comment by SavePointSam** [comment]
My PR to ship `macro.d.ts` has been merged. This issue should be properly resolved with the next release.
